src/imageio/storage/gallery.c

```diff
@@ -175,7 +175,7 @@ void gui_init(dt_imageio_module_storage_t *self)
   g_free(tooltip_text);
 
   widget = dtgtk_button_new(dtgtk_cairo_paint_directory, CPF_DO_NOT_USE_BORDER, NULL);
-  gtk_widget_set_tooltip_text(widget, _("select directory"));
+  gtk_widget_set_tooltip_text(widget, C_("button tooltip", "select directory"));
```

Comment:
"button" ?

Reply:
Won't that be misleading?

Reply:
Why? Would that work for you? Maybe the context is not even needed here? Do you need it?

Reply:
The rationale behind adding context to this and other occurrences of "select directory" is to be able to immediately distinguish (as a translator) whether it's a dialog title or a button or a tooltip. For Estonian translation, the distinction between button and dialog title is important (as the translation should be different), while button and tooltip would share the same translation. So as far as the upcoming Estonian translation is concerned, the contexts "dialog title" (or even just "title") vs. "button/tooltip" (or even just "button") would be fine. I'll change them accordingly.

Reply:
But the issue is that for each different context we will have to translate the text. So that's many many more translation and I really want to avoid that. Translating is already a quite challenging job!

Reply:
From my ca. 15 years experience of translating software, I'd say it's more frustrating to be unable to translate a word/phrase differently in different contexts, than to be forced to translate the same word/phrase two or more times. The latter would just mean picking the previously translated entry from translation memory :)

Reply:
Now changed these tooltips to share the "button" context.